.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,14 +1,29 @@
-Closes #
+### Required checklist
+- [ ] Sample config files updated (both `/etc` folder and `NewDemoConfig` methods) (influxdb and plutonium)
```

Comment:
Can we link @bnpfeife 's issue here so we know when we can remove this?

Reply:
Also can we get real links to the things that need updating (question for @bnpfeife I think)